test/tests/functional/pbs_conf_resv_stale_vnode.py

```diff
@@ -105,6 +105,8 @@ def test_stale_degraded(self):
         Test that a reservation goes into the degraded state
         when one of its vnodes go stale
         """
+        self.server.expect(NODE, {'state': 'free'}, id='vn[0]')
+        self.server.expect(NODE, {'state': 'free'}, id='vnode[0]')
```

Comment:
How about doing this all in one expect() by saying {'state=free': 2} or however many there are.